components/button/button.js

```diff
@@ -41,7 +41,7 @@ class Button extends ButtonMixin(LitElement) {
 
 				button {
 					font-family: inherit;
-					padding: 0.55rem 1.5rem;
+					padding: 0 1.5rem;
```

Comment:
This was necessary otherwise normal buttons ended up being taller than subtle/icon buttons in high contrast mode. The padding isn't needed, since buttons have a `min-height` and use `box-sizing: border-box`.